src/BenchmarkDotNet/Toolchains/Executor.cs

```diff
@@ -66,8 +66,17 @@ private ExecuteResult Execute(Process process, BenchmarkCase benchmarkCase, Sync
             {
                 process.TrySetAffinity(benchmarkCase.Job.Environment.Affinity, logger);
             }
+            Runtime runtime = GetRuntime(benchmarkCase);
 
-            loggerWithDiagnoser.ProcessInput();
+            try
+            {
+                loggerWithDiagnoser.ProcessInput();
+            }
+            catch (System.IO.IOException) when (runtime is WasmRuntime)
```

Comment:
This `catch` block is catching every exception, which means that we could lose some important information (imagine that something else in this method fails)

Reply:
Removed.